public/lessons/responsive_layout/css_grid.md

```diff
@@ -37,8 +37,7 @@
 <p data-height="407" data-theme-id="light" data-slug-hash="ZqvJbM" data-default-tab="css,result" data-user="Dangeranger" data-pen-title="Basic Grid Example" class="codepen">See the Pen <a href="https://codepen.io/Dangeranger/pen/ZqvJbM/">Basic Grid Example</a> by Joshua Burke (<a href="https://codepen.io/Dangeranger">@Dangeranger</a>) on <a href="https://codepen.io">CodePen</a>.</p>
 <script async src="https://static.codepen.io/assets/embed/ei.js"></script>
```

Comment:
check out this script tag and the one on line 13, they are showing up on the page and creating a link starting at https and going through the closing script tag